bootstrap.sh

```diff
@@ -588,7 +590,7 @@ caffeinate -s -w $$ &
 
 install_xcode_cli
 xcode_license
-[ -z "$QUICK" ] && install_homebrew
+! command -v brew &>/dev/null && install_homebrew
```

Comment:
I'm formalizing how to re-test locally and I use `QUICK` in that script, thus, it makes more sense to do this here.